recaf-ui/src/main/java/me/coley/recaf/ui/jfxbuilder/component/container/grid/GridComponent.java

```diff
@@ -0,0 +1,247 @@
+package me.coley.recaf.ui.jfxbuilder.component.container.grid;
+
+import javafx.beans.value.ObservableBooleanValue;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.layout.*;
+import me.coley.recaf.ui.jfxbuilder.NodeToggle;
+import me.coley.recaf.ui.jfxbuilder.component.container.stage.BuildStage;
+import me.coley.recaf.ui.jfxbuilder.component.container.stage.creation.AddingStrategy;
+
+import java.util.Arrays;
+import java.util.function.Consumer;
+
+interface IGridComponent extends GridPaneComponentCreationStage<GridPane, IGridComponent, IGridComponent>,
+		BuildStage<GridPane, IGridComponent>, GridComponentAddStage<GridPane, IGridComponent> {}
+
+public class GridComponent implements IGridComponent {
```

Comment:
Multiple class declarations per file for something that doesn't get exposed feels :suspect: to me, but I see that it does make the declaration in the impl type more clear. If there's a reason that this is the best approach I'm fine with keeping this as-is so long as the reason is documented.